proc_fieldSweep.py

```diff
@@ -0,0 +1,74 @@
+from pylab import *
+from pyspecdata import *
+from pyspecProcScripts import *
+from pyspecProcScripts import lookup_table
+from sympy import symbols
+fl = fl_mod()
+t2 = symbols('t2')
+filter_bandwidth = 20e3
+gamma_eff = (14.903537/3507.48) # MHz / G
+for thisfile,exp_type,nodename,postproc,label_str,freq_slice in [
+        ('220112_150uM_TEMPOL_field_dep_1','ODNP_NMR_comp/field_dependent',
+            'field_sweep_1','field_sweep_v1',
+            'TEMPO field sweep',(-250,250)),
+        ]:
+    s = find_file(thisfile,exp_type=exp_type,expno=nodename)
+    #{{{Obtain ESR frequency and chunk/reorder dimensions
+    v_esr = (s.get_prop('acq_params')['mw_freqs'][0])/1e9
```

Comment:
is this actually what we call nu_B12?

Reply:
(it should be, and I think from context that it is)

Reply:
Yes, I will change the name